cmake/modules/AddSwift.cmake

```diff
@@ -235,7 +238,7 @@ function(_add_variant_link_flags)
     RESULT_VAR_NAME result)
 
   if("${LFLAGS_SDK}" STREQUAL "LINUX")
-    list(APPEND result "-lpthread" "-ldl")
+    list(APPEND result "-lpthread" "-ldl" "-lBlocksRuntime")
```

Comment:
This creates a new dependency for new ports that don't need SourceKit to start bootstrapping; this will also link it into the standard library, which does not need it.  Could we limit this to only SourceKit binaries and libraries?